app/index.html

```diff
@@ -10,6 +10,7 @@
 {{content-for 'tracking-comscore'}}
 {{content-for 'tracking-ua-init'}}
 {{content-for 'head-footer'}}
+{{content-for 'tracking-liftigniter'}}
```

Comment:
We keep the business-critical scripts here. Can we move this just above `{{content-for 'body-footer'}}`?